admin/class-metabox.php

```diff
@@ -33,19 +30,18 @@ public function __construct() {
 	 * the main meta box definition array in the class WPSEO_Meta() as well!!!!
 	 */
 	public static function translate_meta_boxes() {
-		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet Preview', 'wordpress-seo' );
+		self::$meta_fields['general']['snippetpreview']['title'] = __( 'Snippet', 'wordpress-seo' );
```

Comment:
@terw-dan well you did change it in this commit: https://github.com/Yoast/wordpress-seo/commit/1ad37301abab17a2b40ad2a5bb35e00e026dc536

Please revert the change if it was unintended.